.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -48,3 +48,13 @@ body:
       value: kubectl logs -n argo -c wait -l workflows.argoproj.io/workflow=${workflow},workflow.argoproj.io/phase!=Succeeded
     validations:
       required: true
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for submitting this issue! Are you a contributor and if not have you thought about it? 
+
+        Argo Workflows is seeking more community involvement and ultimately more Reviewers and Approvers to help keep it viable. 
+        We are gauging interest in a potential system in which many companies pledge a little bit of time each to help get more people into these roles. 
+        See [here](https://github.com/argoproj/argo-workflows/issues/12229) for more information. 
+        If you think you or your company might be interested, please email julie_vogelman@intuit.com.
```

Comment:
Maybe ask them to comment on the issue instead of sending you an email?

Reply:
that would be more typical - okay, I'll do that